turbo.json

```diff
@@ -5,7 +5,7 @@
       "dependsOn": ["^prebuild"]
     },
     "build": {
-      "dependsOn": ["^build"],
+      "dependsOn": ["^build", "prebuild"],
```

Comment:
nit but prebuild first?

Reply:
I don't think the order matters, since this is just a list of dependencies before this step is run